### PR TITLE
Don't allow "WCA id" in the codebase, prefer "WCA ID".

### PR DESCRIPTION
**Pull request by jfly**
See discussion on https://github.com/thewca/worldcubeassociation.org/issues/268.

### CODE REVIEW

**Comment by jonatanklosko** [comment]
LGTM!

File: config/pre_commit.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "\t" => "tab",
   "\r" => "carriage return",
   "\uFEFF" => "byte order marker (BOM)",
+  ("WCA " + "id") => "We prefer 'WCA ID', see https://github.com/thewca/worldcubeassociation.org/issues/268",
```
Comment (larspetrus) [inline config/pre_commit.rb]: Why `("WCA " + "id")` over `"WCA id"`?

Comment (jfly) [inline config/pre_commit.rb]: One of the funny things about writing code that checks for illegal strings in your codebase... you have to bend over backwards a bit to make sure that you don't actually type any of those illegal strings when you're writing the checking code!

Comment (larspetrus) [inline config/pre_commit.rb]: Aha. How recursive!

**Pull request merged**